lib/promise.js

```diff
@@ -1,6 +1,32 @@
 'use strict';
 
-const Promise = require('bluebird').getNewLibraryCopy();
+Promise.each =  async function(arr, fn) {
+   for(const item of arr) await fn(item);
+}
+
+Promise.prototype.return = func => {
+  return Promise.resolve(func);
```

Comment:
This implementation seems wrong, this would have to be `return this.then(() => func)` also `func` isn't a function?

Reply:
It works as is though.